remixer_ui/src/main/res/layout/remixer_header.xml

```diff
@@ -19,9 +19,19 @@
   xmlns:android="http://schemas.android.com/apk/res/android"
   android:layout_width="match_parent"
   android:layout_height="wrap_content"
+  android:background="@color/headerBackground"
+  android:elevation="6dp"
```

Comment:
Could this and the other hardcoded dimensions be pulled out into dimen resources?